src/types/savedata.ts

```diff
@@ -89,87 +89,88 @@ export interface FullCharacterSaveData
 }
 
 export interface positionUpdate {
-        orientation?: any;
-        frontTilt?: number;
-        sideTilt?: number;
+  orientation?: any;
```

Comment:
modifying this file is pointless since it's generated by https://github.com/QuentinGruber/h1z1-server/blob/master/scripts/genPacketsInterfaces.ts so if you want to fix indent in it you need to modify the script file